src/Components/InputBox.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+import useForm from '../hooks/useForm';
+import useTodoContext from '../hooks/useTodoContext';
+
+const InputBox = () => {
+  const { addNewTodo } = useTodoContext();
```

Comment:
```suggestion
  const { addNewTodo } = useTodo();
```

내부 로직이 context를 이용해서 구현되었다는 점을 네이밍에 굳이 담지 않아도 괜찮을 것 같습니다!